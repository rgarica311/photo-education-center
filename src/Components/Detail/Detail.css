.detailListEl {
  border-bottom: 1px solid lightgrey;
  padding: 15px;
}

.detailListElCol {
  padding: 15px;
  display: flex;
  justify-content: space-evenly;
}

.detailsContainerCol {
  display: flex;
  flex-direction: column;
  justify-content: space-evenly;
  background-color: ghostwhite;
  height: 675px;
  width: 350px;
  padding: 15px;
  padding-left: 0;
  padding-right: 0;
  margin: 0;
  list-style: none;
}

.detailPhoto {
	width: 150px;
	height: 150px;
	background: red;
	-moz-border-radius: 100px;
	-webkit-border-radius: 100px;
	border-radius: 100px;
  position: relative;;
  margin: 0 auto;
  background-image: url('http://creativephotoacademy.com/wp-content/uploads/2016/07/Mark-About-Us-Page.jpg');
  background-size: cover;
}

.datesDetailsRow {
  display: flex;
  top: 50px;
  padding-left: 90px;
  padding-right: 90px;
  position: relative;
  flex-wrap: wrap;
  justify-content: space-between;
}

.beginnerClassDetail {
  position: absolute;
  margin: 0;
  width: 690px;
  top: 550px;
}

.bookButton {
  width: 100px;
  height: 50px;
}
