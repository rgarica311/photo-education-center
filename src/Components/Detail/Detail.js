import React, { Component } from 'react';
import './Detail.css'

export default class Detail extends Component {
  constructor(props){
    super(props);
    this.state = {
    classDesc: props.classDesc,
    title: props.title
    }
  }

  render(){
    console.log('this.state.classDesc', this.state.classDesc)
    return(
      <>
      <ul className='detailsContainerCol'>
        <div className='detailPhoto'></div>
        <li className='detailListEl'>Instructor: Paul's Photo Staff</li>
        
        {
          this.state.classDesc ? <li className='detailListEl'>Description: {this.state.classDesc}</li> : null
        }
        {
          this.state.title ? <li className='detailListEl'>Title: {this.state.title}</li> : null
        }
        <li className='detailListEl'>Venue: Creative Photo Academy at Paul's Photo</li>
        <li className='detailListEl'>Address: 23845 South Hawthorne Blvd., Torrance, California, 90505, United States</li>
        <li className='detailListEl'>Phone: 310-375-7014</li>

      </ul>
      </>
    )
  }
}
