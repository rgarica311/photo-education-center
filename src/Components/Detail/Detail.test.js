import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Detail from './Detail';

describe(`Detail Component`, () => {

  it('renders Detail by default', () => {
    const wrapper = shallow(<Detail />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });

});
