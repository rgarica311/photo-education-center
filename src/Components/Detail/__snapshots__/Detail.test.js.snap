// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Detail Component renders Detail by default 1`] = `
<Fragment>
  <ul
    className="detailsContainerCol"
  >
    <div
      className="detailPhoto"
    />
    <li
      className="detailListEl"
    >
      Instructor: Paul's Photo Staff
    </li>
    <li
      className="detailListEl"
    >
      Venue: Creative Photo Academy at Paul's Photo
    </li>
    <li
      className="detailListEl"
    >
      Address: 23845 South Hawthorne Blvd., Torrance, California, 90505, United States
    </li>
    <li
      className="detailListEl"
    >
      Phone: 310-375-7014
    </li>
  </ul>
</Fragment>
`;
